tests/telemetry/test_telemetry.py

```diff
@@ -60,3 +70,19 @@ def test_telemetry_enabledbydefault(duthost):
         if str(k) == "status":
             status_expected = "enabled";
             pytest_assert(str(v) == status_expected, "Telemetry feature is not enabled")
+
+def test_telemetry_ouput(duthost, ptfhost):
+    """Run pyclient from ptfdocker and show gnmi server outputself.
+    """
+    logger.info('start telemetry output testing')
+    setup_telemetry_forpyclient(duthost)
```

Comment:
For the test case, you may reset the telemetry to the normal mode (client_auth to true )after the test, then if a developer add a new test case, she/he can start from initial state?

Reply:
I have removed reset to normal mode(client auth to true). So adding new test case should not be an issue.

Reply:
what if new test is not through python client? I am ok with either choice. I prefer to set test env after each test.